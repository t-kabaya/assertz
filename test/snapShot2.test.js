const { snap } = require('../index')

snap({ foo: 'foo' }, 'object snap')
