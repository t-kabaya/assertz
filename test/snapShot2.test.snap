[{"snap":{"foo":"foo"},"testName":"object snap","path":"//Users/ipg/kbyProject/helloUnitTest/test/snapShot2.test.js"}]